website/src/pages/showcase/_components/ShowcaseFilterToggle/styles.module.css

```diff
@@ -17,21 +17,22 @@
   border: var(--border) solid var(--ifm-color-primary-darkest);
   cursor: pointer;
   justify-content: space-around;
-  align-items: center;
   opacity: 0.75;
   transition: opacity var(--ifm-transition-fast)
     var(--ifm-transition-timing-default);
   box-shadow: var(--ifm-global-shadow-md);
 }
 
-input:focus ~ .checkboxLabel,
-input:focus-visible ~ .checkboxLabel,
 .checkboxLabel:hover {
   opacity: 1;
   box-shadow: var(--ifm-global-shadow-md),
     0px 0px 2px 1px var(--ifm-color-primary-dark);
 }
 
+input:focus-visible ~ .checkboxLabel::after {
+  outline: 2px solid currentColor;
```

Comment:
I don't know if I prefer solid outline to box shadow. Maybe we just want some more visible shadow?

Reply:
The styling of focus state must be unique, not similar as the hover state. So using an outline is almost always a good idea (at least for a11y).